### DISCUSSION TITLE
I need help with a issue i have

### DISCUSSION THREAD

**Discussion by Ant9cp** [Monika-After-Story/MonikaModDev#8230] [QA - tech support + general]
My game just randomly closes without warning, no error message just nothing 
and it happend before already but, i was trying to figure out if it was my pc's problem or not but i don't know

**Comment by WiiBii2** [comment]
Does it happen immediately after opening the game, or after a few minutes of playing?

**Comment by Ant9cp** [comment]
Somewhere after

**Comment by ThePotatoGuy** [comment]
without additional information, it's hard to say what would be happening. Leaving open as a ts question.
